src/pages/Lending/Pool/Pool.tsx

```diff
@@ -113,15 +113,17 @@ export const Pool = () => {
     null,
   )
   const [repayTxid, setRepayTxid] = useState<string | null>(null)
-  const [collateralAccountId, setCollateralAccountId] = useState<AccountId>(poolAccountId ?? '')
+  const [collateralAccountId, setCollateralAccountId] = useState<AccountId | null>(
```

Comment:
what do we get by using null here and just falling back to `''` everywhere else? both are falsey without all of the extra fallback boilerplate added throughout the codebase.

Reply:
As discussed, `isUtxoAccountId()` was throwing on us in this PR, so rather than doing `Boolean(accountId?.length)` checks everywhere, used the diff size to properly refactor things to *actual* null vs. "`''` to signify null and expect all the places we consume this in handle empty string properly.
Since we're now forced to check things out, now more throwing!